v3/how-to-guides/08-tools/d-subwasm/index.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Verify Runtime Wasm Binaries with Subwasm
+slug: /how-to-guides/v3/tools/subwasm
+keywords: tooling, wasm
+version: 3.0 
+section: how to guides
+category: tools
+---
+
+[Subwasm](https://github.com/chevdor/subwasm) provides a way to expose the features of the runtime for any Substrate chain. 
+This guide steps through the different capabilities that Subwasm provides. 
+
+## Use cases
+
+- Expose verifiable historical information of a chain's runtime for front end applications.
+- Get a quick snapshot of the modules in a runtime.
```

Comment:
I can't suggest direct commits because Lisa's comment bisects the text, but I think we want to get away from **Use cases**. 
Suggest the following:

[Subwasm](https://github.com/chevdor/subwasm) provides a way to expose the features of the runtime for any Substrate chain. This guide shows you how to use Subwasm to do the following: 

- Expose verifiable historical information of a chain's runtime for front end applications.
- Get a snapshot of the modules in a runtime.
- Check for changes between runtimes using a diff.
- Expose a verifiable runtime for transparency.

Reply:
👌🏻